src/native-automation/utils/string.ts

```diff
@@ -14,8 +15,10 @@ export function getResponseAsBuffer (response: GetResponseBodyResponse): Buffer
         : Buffer.from(response.body);
 }
 
-export function toBase64String (str: string): string {
-    return Buffer.from(str).toString('base64');
+export function toBase64String (str: string, contentType = 'charset=utf-8'): string {
```

Comment:
the same thing here. Lets remove default value and make more explicit contentType checking